Fix flaky unit test

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672500728/job/29580106040?pr=658#step:5:866
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672500728/job/29580183775

```
=== RUN   TestInactivityTimerTestSuite/TestTimerIsNotResetAfterDeadline
time="2024-09-02T20:08:08.997039Z" level=info msg="Returning runner due to inactivity timeout" package=runner runner_id=0-MY-DEFAULT-RANDOM-UUID
    inactivity_timer_test.go:84: 
        	Error Trace:	/home/runner/work/poseidon/poseidon/internal/runner/inactivity_timer_test.go:84
        	Error:      	Should be false
        	Test:       	TestInactivityTimerTestSuite/TestTimerIsNotResetAfterDeadline
--- FAIL: TestInactivityTimerTestSuite (3.14s)
    --- PASS: TestInactivityTimerTestSuite/TestResetTimeoutExtendsTheDeadline (0.38s)
    --- PASS: TestInactivityTimerTestSuite/TestRunnerIsNotReturnedBeforeTimeout (0.25s)
    --- PASS: TestInactivityTimerTestSuite/TestRunnerIsReturnedAfterTimeout (0.30s)
    --- PASS: TestInactivityTimerTestSuite/TestSetupTimeoutStopsOldTimeout (0.50s)
    --- PASS: TestInactivityTimerTestSuite/TestStopTimeoutStopsTimeout (0.40s)
    --- PASS: TestInactivityTimerTestSuite/TestTimeoutPassedReturnsFalseBeforeDeadline (0.20s)
    --- PASS: TestInactivityTimerTestSuite/TestTimeoutPassedReturnsTrueAfterDeadline (0.40s)
    --- PASS: TestInactivityTimerTestSuite/TestTimerIsInactiveWhenDurationIsZero (0.30s)
    --- FAIL: TestInactivityTimerTestSuite/TestTimerIsNotResetAfterDeadline (0.40s)
```

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672601360/job/29580373687?pr=662
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672601360/job/29580442933

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672760544/job/29580781602?pr=668
- Failed 2: https://github.com/openHPI/poseidon/actions/runs/10672760544/job/29580843003
- Failed 3: https://github.com/openHPI/poseidon/actions/runs/10672760544/job/29580919548
- Failed 4: https://github.com/openHPI/poseidon/actions/runs/10672760544/job/29580979586
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672760544/job/29581035965

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672757995/job/29580775156
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672757995/job/29580930062

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672909118/job/29581172903
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672909118/job/29581224469

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10672981021/job/29581370164?pr=665
- Successful: https://github.com/openHPI/poseidon/actions/runs/10672981021/job/29581424602

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10679753041/job/29599985911?pr=667
- Successful: https://github.com/openHPI/poseidon/actions/runs/10679753041/job/29602811606

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10681315147/job/29604707683
- Successful: https://github.com/openHPI/poseidon/actions/runs/10681315147/job/29605564049

> same error as above

- Failed: https://github.com/openHPI/poseidon/actions/runs/10682020244/job/29606923642?pr=660
- Failed 2: https://github.com/openHPI/poseidon/actions/runs/10682020244/job/29607516951
- Failed 3: https://github.com/openHPI/poseidon/actions/runs/10682020244/job/29607925108
- Failed 4: https://github.com/openHPI/poseidon/actions/runs/10682020244/job/29608254434
- Successful: https://github.com/openHPI/poseidon/actions/runs/10682020244/job/29608367521

> same error as above

--- a/internal/runner/inactivity_timer_test.go
+++ b/internal/runner/inactivity_timer_test.go
@@ -79,7 +79,7 @@ func (s *InactivityTimerTestSuite) TestTimeoutPassedReturnsTrueAfterDeadline() {
 func (s *InactivityTimerTestSuite) TestTimerIsNotResetAfterDeadline() {
 	time.Sleep(2 * tests.ShortTimeout)
 	// We need to empty the returned channel so Return can send to it again.
-	tests.ChannelReceivesSomething(s.returned, 0)
+	tests.ChannelReceivesSomething(s.returned, tests.ShortTimeout)
 	s.runner.ResetTimeout()
 	s.False(tests.ChannelReceivesSomething(s.returned, 2*tests.ShortTimeout))
 }

--- a/tests/util.go
+++ b/tests/util.go
@@ -16,7 +16,7 @@ import (
 )
 
 // ChannelReceivesSomething waits timeout seconds for something to be received from channel ch.
-// If something is received, it returns true. If the timeout expires without receiving anything, it return false.
+// If something is received, it returns true. If the timeout expires without receiving anything, it returns false.
 func ChannelReceivesSomething(ch chan bool, timeout time.Duration) bool {
 	select {
 	case <-ch:

